ui/app/src/services/configuration.ts

```diff
@@ -95,6 +108,43 @@ export function getPreviewToolsConfig(site: string): Observable<PreviewToolsConf
   );
 }
 
+export function getAudiencesPanelConfig(site: string): Observable<AudiencesPanelConfig> {
+  return getRawContent(site, `/targeting/targeting-config.xml`, 'studio').pipe(
+    map((content) => {
+      try {
+        return JSON.parse(content);
+      } catch (e) {
+        // Not JSON, assuming XML
+        let audiencesPanelConfig: AudiencesPanelConfig = { properties: null };
+        const xml = fromString(content);
+        audiencesPanelConfig.properties = Array.from(xml.querySelectorAll('property')).map((elem) => {
+
+          const name = elem.getElementsByTagName('name')[0].innerHTML,
```

Comment:
1. use querySelector
2 use utils/xml/getInnerHtml 
3 use utils/xml/getInnerHtmlNumber for numeric values